assets/js/no-autoplay.js

```diff
@@ -0,0 +1,44 @@
+window.onload = function() {
```

Comment:
believe all test assets go into this dir `test-assets/...../yourfile.....`  same as the other `mp4` below.